swagger.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"github.com/TykTechnologies/tyk/apidef"
+	"github.com/TykTechnologies/tyk/user"
+)
+
+// parameterBodies
+// swagger:response parameterBodies
+type swaggerParameterBodies struct {
```

Comment:
What it is? and why it is needed?

Reply:
Without this, `go-swagger` won't generate definitions for `apiStatusMessage` and the rest. Thus the swagger spec will be invalid as `"$ref": "#/definitions/APIDefinition"` for example won't work. 

It is more like a way to actually trick `go-swagger` to generating them as definitions in the output file